### DISCUSSION TITLE
Bored

### DISCUSSION THREAD

**Discussion by oM0NG0o** [seerge/g-helper#1004] [Show and tell]
Hye Seerge,

I was bored silly. So I made up some G-Helper Icons for you.
Hope you like and or send me request/ideas.

Mongo

![Untitled](https://github.com/seerge/g-helper/assets/132635900/60ef09cc-28bd-458e-be34-f0a8a97bcfd2)
[Helper-Icons.zip](https://github.com/seerge/g-helper/files/12264704/Helper-Icons.zip)

Part 2 LOL
![Untitled2](https://github.com/seerge/g-helper/assets/132635900/638fbb17-4a72-46c6-bec0-5042f5407c50)

[Helper-icons2.zip](https://github.com/seerge/g-helper/files/12267707/Helper-icons2.zip)

**Comment by Kickskii** [comment]
i prefer the current icon but it would be nice to have an option to have a way to either customise the icon or to have the background color of the icon match with the accent color set in windows / be transparent for a more cleaner look in the taskbar since i have this permamently sitting in the taskbar

![image](https://github.com/seerge/g-helper/assets/56576955/52907a74-4d54-4ebb-8250-9ed083276384)

its probably a low priority item on the to do bucket list but customisation is always nice

**Reply by seerge** [reply]
Icon shows current GPU status (Green - Eco, Blue - Standard, Red - Ultimate), by this ideology background is needed :)

**Comment by oM0NG0o** [comment]
I did same base icon with multi res.
Clean with border.
@seerge As you stated.
32x32
48x48
64x64
72x72
80x80
96x96
128x128
256x256
Think that covers it.

![Untitled](https://github.com/seerge/g-helper/assets/132635900/f2781f61-b30f-4585-a088-504caee96a88)
[Helper-Icos.zip](https://github.com/seerge/g-helper/files/12307394/Helper-Icos.zip)
